{
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:38:30Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5523",
  "aliases": [
    "SNYK-PYTHON-PIPFONTINGLIBARYV1-5932254"
  ],
  "summary": "Malicious code in pipfontinglibaryv1 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pipfontinglibaryv1",
        "purl": "pkg:pypi/pipfontinglibaryv1"
      },
      "versions": [
        "1.1.0",
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PIPFONTINGLIBARYV1-5932254"
    },
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "98c9b6bd6f2c8e26bf19766924410156e3556293ea5edd11e0774155936a4257",
        "import_time": "2024-06-28T02:49:44.780401814Z",
        "id": "RLMA-2024-04305",
        "modified_time": "2024-06-25T13:38:30Z",
        "versions": [
          "1.1.0"
        ]
      },
      {
        "source": "reversing-labs",
        "sha256": "e8b1484a0392f1b2b3d666c6c7d1d2e7d77df0c7b0502d0a9792443644a84075",
        "import_time": "2024-10-24T00:59:38.283382342Z",
        "id": "RLUA-2024-08733",
        "modified_time": "2024-10-16T14:45:55Z",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
